src/main/java/edu/harvard/iq/dataverse/authorization/users/PrivateUrlUser.java

```diff
@@ -12,7 +12,7 @@
  */
 public class PrivateUrlUser implements User {
 
-    public static final String PREFIX = "#";
+    public static final String PREFIX = "!";
```

Comment:
I don't remember why I picked `#` but I think it's fine to change it (I'm glad there's a flyway script).